packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Change
+
+-   Removed a min-width from the `DropdownMenu` component, allowing the menu to accommodate thin contents like vertical tools menus ([#33995](https://github.com/WordPress/gutenberg/pull/33995)).
```

Comment:
Let me know if this sounds good.

Reply:
Sounds good, I was initially confused since this PR adds two min-widths and not removing, but docs are for previous PR :+1: